src/Liquifier.sol

```diff
@@ -204,6 +205,15 @@ contract Liquifier is Initializable, UUPSUpgradeable, OwnableUpgradeable, Pausab
         return eEthShare;
     }
 
+    // Swap user's eETH for Liquifier's stETH
+    function swapEEthForStEth(uint256 _amount) external whenNotPaused nonReentrant {
```

Comment:
why not give permission for liquifier to burn tokens so with don't have that funky eth transfer. Is it a security risk?

Reply:
just burning the tokens will screw up the TVL accounting. This implementation is to mimic the current flow to make sure those accountings alright.